tools/niminst/niminst.nim

```diff
@@ -193,10 +195,14 @@ proc parseCmdLine(c: var ConfigData) =
         var idx = val.find('=')
         if idx < 0: quit("invalid command line")
         c.vars[substr(val, 0, idx-1)] = substr(val, idx+1)
+      of "nim":
+        c.nimExe = val
       else: quit(Usage)
     of cmdEnd: break
   if c.infile.len == 0: quit(Usage)
   if c.mainfile.len == 0: c.mainfile = changeFileExt(c.infile, "nim")
+  # Defaults to `nim` in `PATH` if nothing is specified
+  if c.nimExe.len == 0: c.nimExe = "nim"
```

Comment:
Does this have to takeOS into account and add the exe for widows in the else branch?

Reply:
nope, `cmd.exe` handles that part during `PATH` search